apps/fluent-tester/wdio.conf.windows.js

```diff
@@ -11,7 +11,6 @@ exports.config = {
   specs: [
     'src/E2E/ActivityIndicator/specs/*.win.ts',
     'src/E2E/Button/specs/*.win.ts',
-    'src/E2E/Callout/specs/*.win.ts',
```

Comment:
Please leave a comment in the description for why we are removing this... "Callout not supported by UWP. Removing test" will do :)

Reply:
edited